src/PostgREST/App.hs

```diff
@@ -203,7 +205,7 @@ postgrestResponse conf@AppConfig{..} maybeDbStructure jsonDbS pgVer pool AuthRes
 
   apiRequest <-
     liftEither . mapLeft Error.ApiRequestError $
-      ApiRequest.userApiRequest conf dbStructure req body
+      ApiRequest.userApiRequest conf dbStructure req (BUG.traceShowId body)
```

Comment:
Need help here. Debugging this line shows that the body is empty, does the library ignore it for DELETEs?

Reply:
That looks like [Lazy evaluation](https://wiki.haskell.org/Lazy_evaluation). There will be no trace since the body variable is unused.

You need to add the MutateDelete case here for the body to be passed:

https://github.com/PostgREST/postgrest/blob/fa8dee2f90e82c28f3d390945a275b3025a4cbbb/src/PostgREST/Request/ApiRequest.hs#L312-L318

Reply:
Oops, right. That fixed it.